src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.targets

```diff
@@ -94,6 +94,8 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Condition="'$(OutputType)' == 'WinExe' or '$(OutputType)' == 'Exe'" Include="/ENTRY:$(EntryPointSymbol) /NOEXP /NOIMPLIB" />
       <LinkerArg Include="/NATVIS:&quot;$(MSBuildThisFileDirectory)NativeAOT.natvis&quot;" />
       <LinkerArg Condition="'$(ControlFlowGuard)' == 'Guard'" Include="/guard:cf" />
+      <LinkerArg Condition="('$(OutputType)' == 'WinExe' or '$(OutputType)' == 'Exe') and '$(IlcDefaultStackSize)' == ''" Include="/STACK:0x180000" />
```

Comment:
```suggestion
```
Instead of this, it may be better to set the default value somewhere near the top of the file. `<IlcDefaultStackSize Condition="'$(IlcDefaultStackSize)' == ''">1572864</IlcDefaultStackSize>`

Reply:
Would it make sense to put it in the top `PropertyGroup` (not inside the target), or in one just above this `ItemGroup` (inside the target).

Reply:
I think it can be in the top property group, roughly the same spot as where it is in the Unix targets.

Also, you change the line in the Unix targets to `<IlcDefaultStackSize Condition="`IlcDefaultStackSize` == '' and '$(_linuxLibcFlavor)' == 'musl'">1572864</IlcDefaultStackSize>` while you are on it so that it can be overridden by the user.

(You may want to delay pushing this change until the optional test runs that I have triggered are finished. Otherwise, they will need to be triggered again.)

Reply:
> You may want to delay pushing this change until the optional test runs that I have triggered are finished. Otherwise, they will need to be triggered again.

That's what I was thinking :)
I was going to wait until they're all done before pushing more things